src/language/experimentalOnlineParser/rules.json

```diff
@@ -0,0 +1,938 @@
+{
+  "rules": {
```

Comment:
Please remove `rules` wrapper.

Reply:
We might want to integrate this with ast later on or many include lex rules. So that is why I nested it.

Reply:
@hereisnaman In this case `rules` and `lexerRules` would be confusing so we would need to rename it anyway.
It's a not a stable API so we can always introduce wrapper latter if needed.